Day14/Solution.py

```diff
@@ -1,7 +1,4 @@
 	self.maximumDifference = 0
      
     def computeDifference(self):
-        for a in self.__elements:
-            for b in self.__elements:
-                if abs(a-b) > self.maximumDifference:
-                    self.maximumDifference = abs(a-b)
+        self.maximumDifference=max(a)-min(a)
```

Comment:
@narain1  can you add the code for this whole Difference class in this file as you can see in the screeenshot below.
<img width="809" alt="Screenshot 2019-10-10 at 11 00 31 PM" src="https://user-images.githubusercontent.com/16102594/66592043-e24dd080-ebb1-11e9-96ef-36588c823d78.png">